nexus/users/auth/serializers.py

```diff
@@ -19,9 +20,10 @@ class LoginSerializer(serializers.Serializer):
     password = serializers.CharField(required=True)
 
 
-class RegisterSerializer(serializers.Serializer):
-    email = serializers.EmailField(required=True)
-    password = serializers.CharField(required=True)
+class RegisterSerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
What happens if I just provide `email` and `password` in registration call without all the extra data?

Reply:
In this case a user get registered with that `email` and `password`, and rest all the attributes are given their default values. I have also written a test case to check that registration is successful only with `email` and `password`.